_episodes/06-func.md

```diff
@@ -426,13 +438,15 @@ let's re-define our `offset_mean` function like this:
 
 ~~~
 def offset_mean(data, target_mean_value=0.0):
-    '''Return a new array containing the original data with its mean offset to match the desired value (0 by default).
+    '''Return a new array containing the original data with its mean offset to match the 
```

Comment:
Remove the space at the end of the line and align the next line with `Return a new array`